tensorflow_quantum/python/layers/circuit_executors/expectation_test.py

```diff
@@ -42,7 +45,8 @@ class ExpectationTest(tf.test.TestCase):
 
     def test_expectation_instantiate(self):
         """Test that Expectation instantiates correctly."""
-        expectation.Expectation()
```

Comment:
I think we should also keep the default test `expectation.Expectation()`, and also add `expectation.Expectation(backend=None)` to confirm backwards compatibility

Reply:
Added.